lms/templates/preview_menu.html

```diff
@@ -36,11 +36,13 @@
                             <option value="staff" ${staff_selected}>${_("Staff")}</option>
                             <option value="student" ${student_selected}>${_("Student")}</option>
                             <option value="specific student" ${specific_student_selected}>${_("Specific student")}</option>
-                            % if cohorted_user_partition:
-                                % for group in sorted(cohorted_user_partition.groups, key=lambda group: group.name):
-                                <option value="group.id" data-group-id="${group.id}" ${selected(masquerade_group_id == group.id)}>
-                                    ${_("Student in {content_group}").format(content_group=group.name)}
-                                </option>
+                            % if course_partitions:
```

Comment:
Please add a bok choy testcase for viewing as a learner in an enrollment track. It can go in the existing test_lms_user_preview.py.